docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,172 @@
+
+
+## Creating a `Shoot` cluster
+
+You can wait for the `Seed` to be ready by running
+
+```bash
+kubectl wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed provider-extensions --timeout=5m
+```
+
+Alternatively, you can run `kubectl get seed provider-extensions` and wait for the `STATUS` to indicate readiness:
+
+```bash
+NAME                  STATUS   PROVIDER   REGION         AGE    VERSION      K8S VERSION
+provider-extensions   Ready    gcp        europe-west1   111m   v0.0.0-dev   v1.24.2
+```
+
+In order to create a first shoot cluster, just try one of the following example shoot configurations. Please remember that only shoot cluster with the same provider type as your seed cluster will be started successfully. The others will remain in `<pending>` state.
+
+```bash
+kubectl apply -f ./example/provider-extensions/shoot/aws.yaml
+kubectl apply -f ./example/provider-extensions/shoot/aws-arm64.yaml
+kubectl apply -f ./example/provider-extensions/shoot/azure.yaml
+kubectl apply -f ./example/provider-extensions/shoot/gcp.yaml
+```
+
+You can wait for the `Shoot` to be ready by running `kubectl -n garden-local get shoots` and wait for the `LAST OPERATION` to reach `100%`:
+
+```bash
+NAME        CLOUDPROFILE   PROVIDER   REGION         K8S VERSION   HIBERNATION   LAST OPERATION               STATUS    AGE
+aws         aws            aws        eu-west-1      1.24.3        Awake         <pending>                    healthy   84s
+aws-arm64   aws            aws        eu-west-1      1.24.3        Awake         <pending>                    healthy   65s
+azure       az             azure      westeurope     1.24.2        Awake         <pending>                    healthy   57s
+gcp         gcp            gcp        europe-west1   1.24.3        Awake         Create Processing (43%)      healthy   94s
+```
+
+### Accessing the `Shoot` cluster
+
+Your shoot clusters will have a public DNS entries for their API servers, so that they are could be reached via `kubectl` after you downloaded their `kubeconfig`.
+
+The cluster from the examples listed before have kubeconfigs with static tokens. This is how to access the `gcp` cluster example. The others could be accessed accordingly. 
+
+```bash
+kubectl get secrets -n garden-local gcp.kubeconfig -o yaml | yq .data.kubeconfig | base64 -d > /tmp/kubeconfig-extensions.yaml
+kubectl --kubeconfig=/tmp/kubeconfig-extensions.yaml get nodes
+```
+
+## Deleting the `Shoot` clusters
+
+```bash
+./hack/usage/delete shoot aws garden-local
+./hack/usage/delete shoot aws-arm64 garden-local
+./hack/usage/delete shoot azure garden-local
+./hack/usage/delete shoot gcp garden-local
+```
+
+## Tear down the Gardener environment
+
+Before you delete your local KinD cluster you should shut down your `Shoots` and `Seed` in a clean way to avoid orphaned infrastructure elements in your projects. 
```

Comment:
```suggestion
Before you delete your local KinD cluster, you should shut down your `Shoots` and `Seed` in a clean way to avoid orphaned infrastructure elements in your projects. 
```